content/resources.html

```diff
@@ -55,11 +53,7 @@ <h1 style="clear:left">Resources</h1>
 <h2>Articles About EclEmma</h2>
 
 <ul>
-  <li>Tests f&uuml;r die Tests by Marc R. Hoffmann, <a class="extern" href="http://it-republik.de/jaxenter/eclipse-magazin-ausgaben/Android-000442.html">Eclipse Magazin Vol. 3.2011</a>, 2011/03/25 [de]</li>

```

Comment:
As these paper publications still exist I wouldn't remove them from the list. Just the web links should be removed.